extensions/amp-story/1.0/prerender-active-page.js

```diff
@@ -2,19 +2,26 @@ import {escapeCssSelectorIdent} from '#core/dom/css-selectors';
 import {parseQueryString} from '#core/types/string/url';
 import {getWin} from '#core/window';
 
+/* Cached active page */
+let prerenderActivePage = null;
+
 /**
  * Returns true if the page should be prerendered (for being an active page or
  * first page).
  * @param {!AmpElement} pageElement
  * @return {boolean}
  */
 export function isPrerenderActivePage(pageElement) {
+  if (prerenderActivePage != null) {
+    return prerenderActivePage === pageElement;
```

Comment:
I think it is important for this function to stay pure. As a coding pattern helper functions should really be pure, + storing DOM elements in helpers can super easily lead to memory leaks as it holds hard references and prevents them from being garbage collected.

Can you have either a:
- `isPrerenderActivePage` or something as an `amp-story-page` ivar
- or just store the ID instead of the full page element (I still think it's an anti pattern)